packages/app-degree-pages/src/core/constants/web-api-constants.js

```diff
@@ -6,55 +6,19 @@ const DOMAIN_URL = "https://degrees.apps.asu.edu";
  * @type {import("../types/listing-page-types").ProgramListDataSource}
  */
 const listingPageDefaultDataSource = {
-  endpoint: "https://degrees.apps.asu.edu/t5/service",
-  method: "findAllDegrees",
-  init: "false",
-  fields:
-    `graduateApplyDates,planDeadlines,AsuDegSrchFlg,` +
-    "CollegeAcadOrg,CollegeAcadOrgJoint,DepartmentCode," +
-    "Descr100,Institution,AcadPlan," +
-    "Degree,DegreeDescr,DegreeDescrlong," +
-    "concurrentDegreeMajorMaps,managedOnlineCampus,onlineMajorMapURL," +
-    "AsuCritTrackUrl,AsuOfficeLoc,CampusStringArray,AcadPlan," +
-    "accelerateDegrees,concurrentDegrees," +
-    "CollegeDescr100,CollegeUrl,EmailAddr,DescrlongExtns,AsuProgramFee," +
-    "AsuLangReqFlag,asuAcadpLrfText," +
-    "asuMathReqFlag,additionalMathReqCourse,asuAcadpMrfText,MathIntensity,",
+  endpoint: "https://api.myasuplat-dpl.asu.edu/api/codeset/acad-plans",
+  filter: "activeInDegreeSearch",
+  include: "*",
```

Comment:
This is the default... And I haven't finished reviewing, but if we're not at some point confining the fields to include to only those we're using, that's likely to have a deleterious effect on performance.

Now that the conversion effort is done, can we replace the `*` include here and on line 21 below, with a request that only asks for fields we rely on?